hcs-sxc-java-relay/pom.xml

```diff
@@ -28,14 +28,14 @@
           <version>${artemis-jms-client-all.version}</version>
       </dependency>
     </dependencies>
-    <build>
+<!--    <build>

```

Comment:
should re-enable through profile at a minimum

Reply:
re enablbed